File: src/main/java/com/csvu/FitnessCenter/model/Employee.java
package com.csvu.FitnessCenter.model;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class Employee {
    @Id
    public int emp_id;
    public String emp_fname;
    public String emp_lname;
    public String emp_phone;
    public int fac_id;

    public int getEmp_id() {
        return emp_id;
    }

    public void setEmp_id(int emp_id) {
        this.emp_id = emp_id;
    }

    public String getEmp_fname() {
        return emp_fname;
    }

    public void setEmp_fname(String emp_fname) {
        this.emp_fname = emp_fname;
    }

    public String getEmp_lname() {
        return emp_lname;
    }

    public void setEmp_lname(String emp_lname) {
        this.emp_lname = emp_lname;
    }

    public String getEmp_phone() {
        return emp_phone;
    }

    public void setEmp_phone(String emp_phone) {
        this.emp_phone = emp_phone;
    }

    public int getFac_id() {
        return fac_id;
    }

    public void setFac_id(int fac_id) {
        this.fac_id = fac_id;
    }
}

File: Models/Employee.cs
using System.ComponentModel.DataAnnotations;

namespace Models
{
    public class Employee
    {
        [Key]
        public int Emp_id
        {
            get; set;
        }
        public string Emp_fname
        {
            get; set;
        }
        public string Emp_lname
        {
            get; set;
        }
        public string Emp_phone
        {
            get; set;
        }
        public int Fac_id
        {
            get; set;
        }
    }
}